packages/grafana-ui/src/components/Table/RowExpander.tsx

```diff
@@ -11,13 +11,16 @@ export interface Props {
 }
 
 export function RowExpander({ row, tableStyles }: Props) {
-  return (
-    <div className={tableStyles.expanderCell} {...row.getToggleRowExpandedProps()}>
-      <Icon
-        aria-label={row.isExpanded ? 'Collapse row' : 'Expand row'}
-        name={row.isExpanded ? 'angle-down' : 'angle-right'}
-        size="xl"
-      />
-    </div>
-  );
+  if (row.canExpand) {
```

Comment:
This is more of a nit: What do you think about setting a negated if before the return just to separate the code a bit, I feel like it's more clean/visible, e.g: 
```
if (!row.canExpand) {
   return null
}

return (...)
```